app/assets/stylesheets/addons/_border-color.scss

```diff
@@ -22,5 +22,8 @@
 /// @output `border-color`
 
 @mixin border-color($vals...) {
+  $deprecation-warnings: $output-bourbon-deprecation-warnings;
```

Comment:
For clarity, what about renaming this from `$deprecation-warnings` to `$user-setting-for-deprecation-warnings`?